core/src/apps/ethereum/payment_request.py

```diff
@@ -0,0 +1,23 @@
+from typing import TYPE_CHECKING
+
+from apps.common import writers
+from apps.common.payment_request import PaymentRequestVerifier
+
+if TYPE_CHECKING:
+    from apps.common.payment_request import PaymentRequestType
+
+
+class EthereumPaymentRequestVerifier(PaymentRequestVerifier):
+    @classmethod
+    def get_payment_request_amount(cls, payment_request: PaymentRequestType) -> int | None:  # type: ignore [Variable not allowed in type expression]
+        return (
```

Comment:
Should we check that the `amount` is not larger than 32 bytes?

If so, please add a test.